api.py

```
@@ -0,0 +1,18 @@
+def basic():
+    praisonai = PraisonAI(agent_file="agents.yaml")
+    return praisonai.run()
+
+@app.route('/')
+def home():
+    output = basic()
+    html_output = markdown.markdown(output)
+    return f'<html><body>{html_output}</body></html>'
+
+if __name__ == "__main__":
+    app.run(debug=True)
```

Ruff (0.15.13)

[error] Use of `debug=True` in Flask app detected

(S201)